src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -448,8 +447,20 @@ private void updateLevelButton()
      */
     private void confirmClicked(Button button)
     {
-        MineColonies.getNetwork().sendToServer(new BuildToolPlaceMessage(hutDec.get(hutDecIndex),
-                                                                          getStyles().get(styleIndex), this.pos, rotation, Settings.instance.isInHutMode()));
+        final InventoryPlayer inventory = this.mc.thePlayer.inventory;
```

Comment:
I've pushed a new version that removes the inventory check from the client side and uses a more accurate method to prevent the crash.  

I have also added the chat message to the server.  However, I have also left the chat message on the client side.  

When this crash is prevented on the client side the server code will never run and there would be no feedback to the player if we removed it.  From the players point of view, the crash is because they do not have any huts in their inventory so I did not change the message.